### PR TITLE
Changes for 3.0.1 release

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: ISSUES.txt [hunk 1]
```diff
@@ -1,365 +0,0 @@
-
```
Comment (The-Compiler) [inline ISSUES.txt]: So this was silently dropped with the 3.0.1 release? Are there things in there which are still current and for which we should open issues?

File: doc/en/announce/release-3.0.1.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      Bruno Oliveira
+      Daniel Hahler
+      Dmitry Dygalo
+      Florian Bruhin
+      Marcin Bachry
+      Ronny Pfannschmidt
+      matthiasha
+
+Happy testing,
+The py.test Development Team
```
Comment (The-Compiler) [inline doc/en/announce/release-3.0.1.rst]: #dropthedot :laughing:

File: ISSUES.txt (hunk 1 above)
Comment (nicoddemus) [inline ISSUES.txt]: Hmm TBH I'm not sure, that file is really old (last relevant change was in May 10 2014) so I figured I could delete it. In retrospect I should probably have done that in a separate PR, sorry about that. Should I restore it?

File: doc/en/announce/release-3.0.1.rst (hunk 2 above)
Comment (nicoddemus) [inline doc/en/announce/release-3.0.1.rst]: Hehehe right, gotta remember that for the next release! :grin:

File: ISSUES.txt (hunk 1 above)
Comment (The-Compiler) [inline ISSUES.txt]: Personally I think the best way forward is to open one "Old ISSUES.txt" github issue and then start discussing what's already done, for what we want to open a new issue, and what to drop.

I've only taken a very quick look, but I'd imagine some points there would still be valid and interesting.

Comment (nicoddemus) [inline ISSUES.txt]: Sounds good, done: #1863.
